src/FlashMessage/FlashManager.php

```diff
@@ -22,27 +22,12 @@
  */
 final class FlashManager implements FlashManagerInterface
 {
-    private RequestStack $requestStack;
-
-    /**
-     * @var array<string, array<string>>
-     */
-    private array $types;
-
-    /**
-     * @var array<string, string>
-     */
-    private array $cssClasses;
-
     /**
      * @param array<string, array<string>> $types      Sonata flash message types array (defined in configuration)
      * @param array<string, string>        $cssClasses Css classes associated with $types
      */
-    public function __construct(RequestStack $requestStack, array $types, array $cssClasses)
+    public function __construct(private RequestStack $requestStack, private array $types, private array $cssClasses)
```

Comment:
For construct with more than 1 argument , I have been doing multiline, wdyt? for readability

Reply:
Sure done